deploy/hypershift-obo-alertmanager-config/config.yaml

```diff
@@ -0,0 +1,12 @@
+deploymentMode: "SelectorSyncSet"
+SelectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
```

Comment:
I think we need to deploy this on management cluster only, you can refer to the other one for ocm-agent  SSS.

Reply:
@bmeng - You mean only keep the key section below? 
```
key: api.openshift.com/managed
```
I saw both `ocm-agent-operator-managedfleetnotifications` and `ocm-agent-operator-managednotifications` have SSS with the `fedramp` only which indicates U.S. government-wide clusters?
```
- key: api.openshift.com/fedramp                                                                
```

Reply:
to deploy on management cluster only, we need https://github.com/openshift/ocm-agent/blob/master/hack/00-ocm-agent.selectorsyncset.yaml.tmpl#L49